Userland/Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp

```diff
@@ -329,7 +329,11 @@ class TinyVGReader {
 
 ErrorOr<NonnullRefPtr<TinyVGDecodedImageData>> TinyVGDecodedImageData::decode(Stream& stream)
 {
-    auto header = TRY(decode_tinyvg_header(stream));
+    return decode(stream, TRY(decode_tinyvg_header(stream)));
```

Comment:
Can't we wait [this](https://discord.com/channels/830522505605283862/834851530946773022/1127001484677099641) refactor before landing this commit?

The error state management brings some complexity that I would like to be spared.

Reply:
You could also cherry-pick this change into your refactor since there's yet to be a PR for it :)

Reply:
Sounds good, I got my local copy!